test/test_api/test_api.py

```diff
@@ -9,6 +9,13 @@
 from pytest import raises
 
 
+def _clear_last_exception_fields_in_sys():
```

Comment:
Perhaps easier, test (in python 2.6, interestingly not in 3.4) also fails in linter HEAD when skipping all integration tests:

```
diff --git a/test/run.py b/test/run.py
index 6be9b5f..03e97e8 100755
--- a/test/run.py
+++ b/test/run.py
@@ -135,7 +135,7 @@ class IntegrationTestCase(object):
         self.start = start
         self.line = line
         self.path = path
-        self.skip = None
+        self.skip = "Skip all tests"

     @property
     def module_name(self):
```